### PR TITLE
Notes

### PR DESCRIPTION
**Pull request by brittany-johnson**
note styling for baked PDF and note styling for intro to business

### CODE REVIEW

File: styles/themes/theme3/style/chapter-title.scss [hunk 1]
```diff
@@ -4,7 +4,8 @@
  */
 [data-type="chapter"] {
   > h1[data-type="document-title"] {
```
Comment (helenemccarron) [inline styles/themes/theme3/style/chapter-title.scss]: should this change take place in this PR? It feels like the splash image is using a very specific header size, we might need to test this separately

File: styles/framework/mixins/_notes.scss [hunk 2]
```diff
@@ -7,12 +7,16 @@
...
 //TODO: notes inside of notes
 
+
 @mixin baseNoteStyle() {
   width: 100%;
   box-decoration-break: slice;
   box-sizing: border-box;
   margin-bottom: size-scale(1);
 
+  h3[data-type="title"] {
```
Comment (helenemccarron) [inline styles/framework/mixins/_notes.scss]: should this be handled in the subtitle mixin? It's a bit confusing to have a title defined in the base style when there is a `baseNoteTitle` mixin and a `noteSubtitle`

File: styles/framework/mixins/_notes.scss [hunk 3]
```diff
@@ -22,73 +26,97 @@
       @include link-style(settings(note-link));
     }
   }
+  .os-note-body {
+    line-height: line-height(1);
```
Comment (helenemccarron) [inline styles/framework/mixins/_notes.scss]: should those values be moved to settings?

File: styles/framework/mixins/_notes.scss [hunk 4]
```diff
@@ -22,73 +26,97 @@
       @include link-style(settings(note-link));
     }
   }
+  .os-note-body {
```
Comment (helenemccarron) [inline styles/framework/mixins/_notes.scss]: I wonder if we want to isolate the body in it's own mixin, like the titles then call it with the proper colors (something similar to how we broke down the tables in reusable mixins? We could it it l.86 and l.115

File: styles/framework/mixins/_notes.scss [hunk 5]
```diff
@@ -22,73 +26,97 @@
...
     border-top-right-radius: settings(note-top-border-radius); //todo: radius variables
     border-top-left-radius: settings(note-top-border-radius);
-
     @if $image {
       background: url($image) $titleBGcolor no-repeat right;
+      background-size: contain;
     }
 
   }
   .os-note-body {
```
Comment (helenemccarron) [inline styles/framework/mixins/_notes.scss]: see comment about `.os-note-body` mixin

File: styles/framework/mixins/_notes.scss [hunk 6]
```diff
@@ -22,73 +26,97 @@
...
 
-    border-top: settings(note-TopBar-border-radius) $topBorderColor;
+    border-top: settings(note-TopBar-border-radius) settings(note-TopBar-border) $topBorderColor;
     border-bottom:  settings(note-border) $borderColor;
     border-top-right-radius: settings(note-TopBar-border-radius);
     border-top-left-radius: settings(note-TopBar-border-radius);
-    padding: settings(note-title-padding);
-    //width: @note-border-top-width;
   }
   .os-note-body {
```
Comment (helenemccarron) [inline styles/framework/mixins/_notes.scss]: see comment about `.os-note-body` mixin

File: styles/framework/_import-config.scss [hunk 7]
```diff
@@ -1,6 +1,7 @@
+@import "/Users/bj20/openstax/cnx-recipes/node_modules/normalize.css/normalize.css";
```
[the remark this answers is not in the document]
Comment (brittany-johnson) [reply styles/framework/_import-config.scss]: OOPS! That was not intentional

File: styles/framework/mixins/_notes.scss (hunk 2 above)
Comment (brittany-johnson) [reply styles/framework/mixins/_notes.scss]: Ahh yes. I'll move this to the `noteTitle` mixin since the note titles are `h3` not the subtitles

**Comment by brittany-johnson** [comment]
@helenemccarron Making a note of this before I merge and forget to mention it: I merged `svg-and-eoc` branch into this branch because that branch contains the code that allows functions to be called in the settings file and for those settings to compile properly when called.

**Comment by brittany-johnson** [comment]
All code can be found in https://github.com/Connexions/cnx-recipes/pull/433 . Closing this PR to reduce duplication of work

**Pull request closed without merging**